fishtest/fishtest/rundb.py

```diff
@@ -647,3 +653,20 @@ def update_spsa(self, worker, run, spsa_results):
 
     if grow_summary:
       spsa['param_history'].append(summary)
+
+@jit(nopython=True)
+def jit_spsa_param_clip_round(theta, pmin, pmax, increment, clipping):
+  value = 0.0
```

Comment:
@tomtor IMO the initialization can be dropped

Reply:
Yes, you are correct. Changes should be applied to original `spsa_param_clip_round` around line 520 because for some reason the JIT compiler didn't speed-up.